presto-main/src/test/java/com/facebook/presto/sql/query/TestCurrentUser.java

```diff
@@ -27,7 +27,8 @@
     public void testCurrentUser()
     {
         Session session = testSessionBuilder().setIdentity(new Identity("test_current_user", Optional.empty())).build();
-        QueryAssertions queryAssertions = new QueryAssertions(session);
-        queryAssertions.assertQuery("SELECT CURRENT_USER", "SELECT CAST('" + session.getUser() + "' AS VARCHAR)");
+        try (QueryAssertions queryAssertions = new QueryAssertions(session)) {
```

Comment:
Can you provide a static method `QueryAssertions.assertQuery` to handle such simple cases.

Reply:
I don't want to mix static and non-static methods (with similar or same names) in one class. Let me abstain from doing this for now.